INN/umbrella-citylimits
# Update ads spacing, font size, background color

## Changes

This pull request makes the following changes:

- Removes tan background color from ads
- Changes `Advertisement` text to `font-size: 10px;`
- Reduces padding on ads from 5vw to 2.5vw

![Screen Shot 2020-02-04 at 1 17 11 PM](https://user-images.githubusercontent.com/18353636/73773833-e39e9c80-4750-11ea-8a21-f3ea15a2187a.png)

![Screen Shot 2020-02-04 at 1 17 18 PM](https://user-images.githubusercontent.com/18353636/73773832-e39e9c80-4750-11ea-8f07-1d60ef56670a.png)

## Why



For #145

## Testing/Questions

Features that this PR affects:

- Ads


Questions that need to be answered before merging:


Steps to test this PR:

1. View an ad on the header or in an article and verify the new font size, spacing, and background look ok.
